composer.json

```diff
@@ -20,7 +20,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.4 || ~8.0.0",
```

Comment:
Note: Travis-CI is dead anyway, so Github Actions **ABSOLUTELY MUST** be enabled before merging this patch.

Reply:
`CHANGELOG.md` entries for the entire patch and this line change are needed.

Reply:
I think that the note in the patch shouldn't be that we're bumping to 7.4 because 7.3 is unsupported (it is actually still supported; it's just getting security updates **only** for the coming year), but because _you are making use of PHP 7.4+ features_ (specifically, arrow functions and typed properties).

Reply:
Works for me - I generally just need to get some speed on this package, and the various outdated approaches taken to types make it very fragile to work with, and reduce dev speed substantially :|